DSharpPlus/Entities/Interaction/Application/DiscordApplicationOAuth2InstallParams.cs

```diff
@@ -0,0 +1,27 @@
+namespace DSharpPlus.Entities;
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
```

Comment:
namespace below usings (also, ideally different using prefixes get a newline between them, but v5 doesn't do that half the time)